src/main/java/com/commercetools/sync/categories/utils/CategoryUpdateActionUtils.java

```diff
@@ -0,0 +1,212 @@
+                newCategoryName,
+                ChangeName.of(newCategoryName));
+    }
+
+    /**
+     * Compares the {@link LocalizedString} slugs of a {@link Category} and a {@link CategoryDraft} and returns a
+     * {@link SyncResult<Category>} containing a list of update actions, which would contain the "changeSlug"
+     * {@link UpdateAction<Category>}. If no update action is needed, for example in case where both the {@link Category}
+     * and the {@link CategoryDraft} have the same slugs, the sync result would have an empty list of update actions.
+     *
+     * @param oldCategory the category which should be updated.
+     * @param newCategory the category draft where we get the new slug.
+     * @return A sync result with a list containing the update action or a sync result containing an empty list
+     * of update actions if the slugs are identical.
+     */
+    @Nonnull
+    public static SyncResult<Category> buildChangeSlugUpdateAction(
+            @Nonnull final Category oldCategory,
+            @Nonnull final CategoryDraft newCategory) {
+        final LocalizedString newCategorySlug = newCategory.getSlug();
+        return buildUpdateAction(oldCategory.getSlug(),
+                newCategorySlug,
+                ChangeSlug.of(newCategorySlug));
+    }
+
+    /**
+     * Compares the {@link LocalizedString} descriptions of a {@link Category} and a {@link CategoryDraft} and returns a
+     * {@link SyncResult<Category>} containing a list of update actions, which would contain the "setDescription"
+     * {@link UpdateAction<Category>}. If no update action is needed, for example in case where both the {@link Category}
+     * and the {@link CategoryDraft} have the same description values, the sync result would have an empty list of
+     * update actions.
+     * <p>
+     * Note: If the description of the new {@link CategoryDraft} is null, a message is added in the statistics
+     * {@link java.util.Map} of the {@link SyncResult<Category>}.
+     *
+     * @param oldCategory the category which should be updated.
+     * @param newCategory the category draft where we get the new description.
+     * @return A sync result with a list containing the update action or a sync result containing an empty list
+     * of update actions if the description values are identical.
+     */
+    @Nonnull
+    public static SyncResult<Category> buildSetDescriptionUpdateAction(
+            @Nonnull final Category oldCategory,
+            @Nonnull final CategoryDraft newCategory) {
+        final LocalizedString newCategoryDescription = newCategory.getDescription();
+        if (newCategoryDescription != null) {
+            return buildUpdateAction(oldCategory.getDescription(), newCategoryDescription,
+                    SetDescription.of(newCategoryDescription));
+        } else {
+            return SyncResult.ofStatistic(SyncMessages.CATEGORY_SET_DESCRIPTION_EMPTY_DESCRIPTION);
```

Comment:
Let's say I want to use the sync library and I want to remove the description set on a category. So my source data would have the description of this category not set. 

I think it wouldn't be harmful, If I get to know that the category wasn't updated as I intended _as opposed to_ not getting any notification about it and later on discover that the category wasn't updated and then have to look up in the API docs why it didn't happen.